MAUI/DataGrid/columns.md

```diff
@@ -0,0 +1,205 @@
+
+{% tabs %}
+{% highlight xaml %}
+<syncfusion:SfDataGrid x:Name="sfDataGrid" AutoGenerateColumnsMode="None" ItemsSource="{Binding OrderInfoCollection}" >
+    <syncfusion:SfDataGrid.Columns>
+        <syncfusion:DataGridNumericColumn HeaderText="Order ID" MappingName="OrderID"/>
+        <syncfusion:DataGridTextColumn  HeaderText="Customer ID" MappingName="CustomerID"/>
+        <syncfusion:DataGridTextColumn  HeaderText="Customer" MappingName="Customer"/>
+        <syncfusion:DataGridTextColumn  HeaderText="Ship City" MappingName="ShipCity"/>
+        <syncfusion:DataGridTextColumn  HeaderText="Ship Country" MappingName="ShipCountry"/>
+    </syncfusion:SfDataGrid.Columns>
+</syncfusion:SfDataGrid>
+{% endhighlight %}
+
+{% highlight c# %}
+this.sfDataGrid.AutoGenerateColumnsMode = AutoGenerateColumnsMode.None;
+
+DataGridNumericColumn orderIdColumn = new DataGridNumericColumn { HeaderText = "Order ID", MappingName = "OrderID" };
+DataGridTextColumn customerIdColumn = new DataGridTextColumn { HeaderText = "Customer ID", MappingName = "CustomerID" };
+DataGridTextColumn customerColumn = new DataGridTextColumn { HeaderText = "Customer", MappingName = "Customer" };
+DataGridTextColumn shipCityColumn = new DataGridTextColumn { HeaderText = "Ship City", MappingName = "ShipCity" };
+DataGridTextColumn shipCountryColumn = new DataGridTextColumn { HeaderText = "Ship Country", MappingName = "ShipCountry" };
+
+this.sfDataGrid.Columns.Add(orderIdColumn);
+this.sfDataGrid.Columns.Add(customerIdColumn);
+this.sfDataGrid.Columns.Add(customerColumn);
+this.sfDataGrid.Columns.Add(shipCityColumn);
+this.sfDataGrid.Columns.Add(shipCountryColumn);
+{% endhighlight %}
+{% endtabs %}
+
+## Column manipulation
+
+You can get the columns from the `SfDataGrid.Columns` property.
+
+### Adding column to DataGrid
+
+You can add a column to the DataGrid at runtime by adding instance of a `DataGridColumn` to the `SfDataGrid.Columns` collection.
+
+{% tabs %}
+{% highlight c# %}
+this.sfDataGrid.Columns.Add(new DataGridTextColumn() { HeaderText = "Order ID", MappingName = "OrderID" });
+{% endhighlight %}
+{% endtabs %}
+
+### Accessing column
+
+You can access the column through its column index or [DataGridColumn.MappingName](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridColumn.html#Syncfusion_Maui_DataGrid_DataGridColumn_MappingName) from the `SfDataGrid.Columns` collection.
+
+{% tabs %}
+{% highlight c# %}
+DataGridColumn column = this.sfDataGrid.Columns[1];
+// OR
+DataGridColumn column = this.sfDataGrid.Columns["OrderID"];
+{% endhighlight %}
+{% endtabs %}
+
+### Clearing or removing a column
+
+You can remove all the columns by clearing the `SfDataGrid.Columns` property.
+
+{% tabs %}
+{% highlight c# %}
+this.sfDataGrid.Columns.Clear();
+{% endhighlight %}
+{% endtabs %}
+
+You can remove a column using `Remove` and `RemoveAt` methods.

```

Comment:
You can remove a column using the `Remove` and `RemoveAt` methods.

Reply:
Modified.